internal/provider/slickvpn/updater/website.go

```diff
@@ -47,18 +57,18 @@ type serverData struct {
 	city    string
 }
 
-var (
-	locationTableSelector = cascadia.MustCompile(`table#location-table > tbody > tr`) //nolint:gochecknoglobals
-	ovpnHrefSelector      = cascadia.MustCompile(`a[href$='.ovpn']`)                  //nolint:gochecknoglobals
-)
+func parseHTML(rootNode *html.Node) (hostToData map[string]serverData, err error) {
+	locationTableNode := htmlutils.GetFirstNodeByID(rootNode, "location-table")
+	if locationTableNode == nil {
+		return nil, fmt.Errorf("unable to find html node with matching id")
```

Comment:
Please use a sentinel error (aka global error variable `var ErrXXX = errors.New("...")`), more context at https://qqq.ninja/blog/post/wrap-go-errors-like-it-s-christmas/

Same for `fmt.Errorf` below on line 68 😉